pkg/flag/vm_flags.go

```diff
@@ -0,0 +1,50 @@
+package flag
+
+var (
+	awsRegionFlag = Flag[string]{
+		Name:       "region",
+		ConfigName: "cloud.aws.region",
```

Comment:
perhaps we want to change configName.
e.g.:
```suggestion
		ConfigName: "vm.region",
```